tests/test-cases/Group23-VIC-Machine-Service/23-02-VCH-List.robot

```diff
@@ -120,3 +122,15 @@ Get VCH List Within Invalid Datacenter and Compute Resource
 
     Verify Return Code
     Verify Status Not Found
+
+Get Empty VCH List When No VCH deployed
+    Cleanup VIC Appliance On Test Server
+
+    Get VCH List
+
+    Verify Return Code
+    Verify Status Ok
+
+    Verify VCH List Empty
+
+    [Teardown]  NONE
```

Comment:
I just removed this test here:
https://github.com/vmware/vic/pull/7879/files

Please do not add it back in.  It inherently cannot be run in our CI, it needs to be moved to manual tests

Reply:
oh sorry I didn't see it. It got automatically added by git merge. will remove it now.

Reply:
just removed